kernel-install: add page (#10919)
tldr-pages/tldr: pages/linux/kernel-install.md

# kernel-install

> > Add and remove kernel and initrd images to and from /boot.

Author: Change added.

> > More information: <https://man.archlinux.org/man/kernel-install.8>.

Reviewer: ```suggestion
> More information: <https://manned.org/kernel-install.8>.
```
We use Manned as the default fallback when there isn't an author's documentation. (And this command isn't an Arch Linux project so we can use manned)

Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#more-information-links

Author: Change added.

- Add kernel and initramfs images to bootloader partition:

> `sudo kernel-install add {{kernel-version}} {{kernel-image}} {initrd-file ...}`

Reviewer 2: ```suggestion
`sudo kernel-install add {{kernel-version}} {{kernel-image}} {{initrd-file ...}}`
```

Reviewer: Seem like we both reviewed at same time 😅 , will resolve this suggestion alone. The other one LGTM.

> `sudo kernel-install add {{kernel-version}} {{kernel-image}} {initrd-file ...}`

Author: Change added.

- Remove kernel from the bootloader partition:

`sudo kernel-install remove {{kernel-version}}`

> - Show various path and parameter configured or auto-detected:

Reviewer 2: ```suggestion
- Show various paths and parameters that have been configured or auto-detected:
```

Author: Change added.
